### ISSUE TITLE
eventd/eventd-git 0.27.0-1 fails to build with meson 0.56.0-1 on Arch

### ISSUE THREAD

**Issue by klauswaugh** [sardemff7/eventd#50]
I'm trying to install the latest version of eventd from the AUR but it fails to build with the following message: 

```
plugins/nd/meson.build:86: WARNING: Subproject name has a path separator. This may cause unexpected behaviour.

plugins/nd/meson.build:85:4: ERROR: Neither a subproject directory nor a libgwater/xcb.wrap file was found.

A full log can be found at /home/jed/.cache/yay/eventd/src/build/meson-logs/meson-log.txt
```
I've uploaded the full [meson-log.txt](https://github.com/sardemff7/eventd/files/5585685/meson-log.txt) file in case it's of any help.

A cursory google search brings up a similar [issue](https://github.com/davatorium/rofi/issues/1159) with rofi and meson 0.55 back in August that I think may be to blame here. I'm not very familiar with meson and I was wondering if there was going to be a fix at some point the future or if somebody could suggest a quick fix to get eventd to build?

**Comment by limansky** [comment]
Same on Gentoo with meson 0.59.4

**Comment by sardemff7** [comment]
Should be fixed in 7c29f09efdc8de4c560f277ac95c066acca1317e for Git builds, can you confirm?
Sadly had more (paid) work lately and too little time for a release, as I package it myself in many formats and it takes time to test them properly

**Comment by limansky** [comment]
@sardemff7 U're welcome. Any plans to prepare a new release?
